CHANGELOG.md

```diff
@@ -0,0 +1,162 @@
+- `children_changed` now always includes layout and paint request. ([#839] by [@xStorm])
+- Mostly complete Wasm backend for druid-shell. ([#759] by [@elrnv])
+- `UpdateCtx::submit_command`. ([#855] by [@cmyr])
+- `request_paint_rect` for partial invalidation. ([#817] by [@jneem])
+- Window title can be any `LabelText` (such as a simple `String`). ([#869] by [@cmyr])
+- `Label::with_font` and `set_font`. ([#785] by [@thecodewarrior])
+
+### Changed
+
+- Renamed `Split` constructors to `Split::rows` and `columns`. ([#738] by [@xStorm])
+- `Split::splitter_size` no longer includes padding. ([#738] by [@xStorm])
+- Renamed `Event::MouseMoved` to `MouseMove`. ([#825] by [@teddemunnik])
+- `has_focus` no longer returns false positives. ([#819] by [@xStorm])
+- `Event::Internal(InternalEvent)` now bundles all internal events. ([#833] by [@xStorm])
+- `WidgetPod::set_layout_rect` now requires `LayoutCtx`, data and `Env`. ([#841] by [@xStorm])
+- `request_timer` now uses `Duration` instead of `Instant`. ([#847] by [@finnerale])
+
+### Deprecated
+
+- Nothing
+
+### Removed
+
+- The optional GTK feature for non-Linux platforms. ([#611] by [@pyroxymat])
+
+### Fixed
+
+- GTK: Use the system locale. ([#798] by [@finnerale])
+- GTK: Actually close windows ([#797] by [@finnerale])
+- Windows: Respect the minimum window size. ([#727] by [@teddemunnik])
+- Windows: Respect resizability. ([#712] by [@teddemunnik])
+- Windows: Capture mouse for drag actions. ([#695] by [@teddemunnik])
+- Start focus cycling from non-registered-for-focus widgets. ([#819] by [@xStorm])
+- Propagate `Event::FocusChanged` to focus gaining widgets as well. ([#819] by [@xStorm])
+- GTK: Prevent crashing on pop-ups. ([#837] by [@finnerale])
+- Keep hot state  consistent with mouse position. ([#841] by [@xStorm])
+- Open file menu item works again. ([#851] by [@kindlychung])
+- Supply correct `LifeCycleCtx` to `Event::FocusChanged`. ([#878] by [@cmyr])
+- Windows: Termiate app when all windows have closed. ([#763] by [@xStorm])
+
+### Visual
+
+- Improved `Split` accuracy. ([#738] by [@xStorm])
+- Build-in widgets no longer stroke outside their `paint_rect`. ([#861] by [@jneem])
+
+### Docs
+
+- Reduce the flashing in ext_event and identity examples. ([#782] by [@futurepaul])
+- `Env` got a new example and usage hints. ([#796] by [@finnerale])
+- Usage of bloom filters got documented. ([#818] by [@xStorm])
+- Book chapters about `Painter` and `Controller` were added. ([#832] by [@cmyr])
+- Added hot glow option to multiwin example. ([#845] by [@xStorm])
+
+### Maintenance
+
+- Replace `#[macro_use]` with normal `use`. ([#808] by [@totsteps])
+- Clippy checks are now enabled for all targets. ([#850] by [@xStorm])
+- Added rendering tests. ([#784] by [@fishrockz])
+- CI testing has been revamped. ([#857] by [@xStorm])
+- Associate timers with widget ids. ([#831] by [@sjoshid])
+- Add a changelog. ([#889] by [@finnerale])
```

Comment:
I don't think future tense makes sense for a changelog. So this should be *Added* not *Add*. Should probably check other lines too. Generally speaking I guess both past tense (*Added a changelog*) and present tense (*There is now a changelog*) are fine and both can be used, but yeah not future.

Reply:
Yea I thing I've mixed tenses a bit :sweat_smile: 
I'm not quite sure what fits better (present or past tense).